packages/manager/modules/pci/src/projects/project/kubernetes/details/audit-logs/audit-logs.html

```diff
@@ -0,0 +1,45 @@
+<div>
+    <span data-translate="kube_logs_description"></span>
+    <a
+        data-ng-href="{{::$ctrl.getLogsInfoLink()}}"
+        target="_blank"
+        rel="noopener"
+    >
+        <span data-translate="kube_logs_description_link"></span>
+        <span class="oui-icon oui-icon-external_link" aria-hidden="true"></span>
+    </a>
+
+    <tail-logs>
+        <div
+            data-ng-repeat="log in $ctrl.logger.logs track by $index"
+            data-ng-class="{
+            'text-warning' : log.message.audit_responseStatus >= 300 && log.message.audit_responseStatus < 499,
```

Comment:
Maybe use constants instead of maintaining values in HTML

Reply:
I think is not required, i had a single usage.
Thanks

Reply:
From my personal knowledge and practices. We create a function:

1. When this function is used by many other codes(generic). so it's not my case.
2. When the logic is complex to be understandable. i think is not the case

@JeremyDec, may be i forgot something ?

Reply:
@mohammed-zahaf, To be nitpicking , 499 is used twice :) 
Even though it is a single usage, it is easier to update a constants file than some values lost in HTML file, also it is more meaningful to give names to those value and we could be able to document easily what those values represent

Reply:
thanks @marie-j, by 'single usage' i mean, i don't need those values outside of `ng-class`.
I used them only for only here not everywhere.
So, we will create a const and maybe functions but for a minor usage.

Reply:
Changed.